filesystem/iso9660.ksy

```diff
@@ -216,60 +94,846 @@ types:
+                                type: b1
+                              - id: other_del
+                                doc-ref: rras 6c.4
+                                type: b1
+                              - id: group_read
+                                doc-ref: rras 6c.3
+                                type: b1
+                              - id: group_write
+                                doc-ref: rras 6c.2
+                                type: b1
+                              - id: group_exec
+                                doc-ref: rras 6c.1
+                                type: b1
+                              - id: group_del
+                                doc-ref: rras 6c.0
+                                type: b1
+                              - id: reserved
+                                doc-ref: rras 6d.7
+                                type: b1
+                              - id: exec_script
+                                doc-ref: rras 6d.6
+                                type: b1
+                              - id: reent_exec
+                                doc-ref: rras 6d.5
+                                type: b1
+                              - id: archived
+                                doc-ref: rras 6d.4
+                                type: b1
+                              - id: owner_read
+                                doc-ref: rras 6d.3
+                                type: b1
+                              - id: owner_write
+                                doc-ref: rras 6d.2
+                                type: b1
+                              - id: owner_exec
+                                doc-ref: rras 6d.1
+                                type: b1
+                              - id: owner_del
+                                doc-ref: rras 6d.0
+                                type: b1
+                      susp_ce:
+                        doc-ref: susp 5.1
+                        seq:
+                          - id: length
+                            type: u1
+                          - id: version
+                            contents: [ 0x1 ]
+                          - id: ca_location
+                            type: u4bi
+                          - id: ca_offset
+                            type: u4bi
+                          - id: ca_length
+                            type: u4bi
+                      rrip_cl:
+                        doc-ref: rrip 4.1.5.1
+                        seq:
+                          - id: length
+                            type: u1
+                          - id: version
+                            contents: [ 0x1 ]
+                          - id: lba_child
+                            type: u4bi
+                      susp_er:
+                        doc-ref: susp 5.5
+                        seq:
+                          - id: length
+                            type: u1
+                          - id: version
```

Comment:
```yaml
type: u1
valid: 1
```

Reply:
The length is not always 1, but I fixed the "version" with the same logic